docs(troubleshooting): add network stalled solution for ubuntu linux (fixes #11468) (#11959)
vitejs/vite: docs/guide/troubleshooting.md

- /etc/systemd/system.conf
- /etc/systemd/user.conf

> For Ubuntu Linux the working solution is to add the line `* - nofile 65536` to the file `/etc/security/limits.conf` instead of updating systemd config files.

Reviewer: ```suggestion
For Ubuntu Linux, you may need to add the line `* - nofile 65536` to the file `/etc/security/limits.conf` instead of updating systemd config files.
```

Maybe this should be phrased as "you may need to" as another alternative if the above doesn't work for them.

---

Anyways, I don't have a way to test this so I'd have to trust your solution, which would be good anyways if it helps someone 😄
